mix.exs

```diff
@@ -37,7 +37,10 @@ defmodule Tesla.Mixfile do
     [
       maintainers: ["Tymon Tobolski"],
       licenses: ["MIT"],
-      links: %{"GitHub" => "https://github.com/elixir-tesla/tesla"}
+      links: %{
+        "GitHub" => @source_url,
+        "Changelog" => "#{@source_url}/releases"
```

Comment:
After https://github.com/elixir-tesla/tesla/pull/728 is merged, we will have a CHANGELOG file